control-plane-operator/controllers/hostedcontrolplane/v2/assets/openshift-apiserver/deployment.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         livenessProbe:
           failureThreshold: 3
           httpGet:
-            path: healthz
+            path: livez?exclude=etcd
```

Comment:
We need to drop the `initialDelaySeconds` to `0` since there is a startup probe.

Reply:
@Evan-Reilly ping in case you missed this.